doc/vim-lsp.txt

```diff
@@ -224,6 +224,11 @@ g:lsp_preview_keep_focus                         *g:lsp_preview_keep_focus*
 	let g:lsp_preview_keep_focus = 0
 <
     * |preview-window| can be closed using the default vim mapping - `<c-w><c-z>`.
+    * preview-window can be also automatically closed after completion with
```

Comment:
need to include `|preview-window|` with pipe so can navigate easily to more details about preview window.

Reply:
Fixed. Btw. can you please send me a link to the VIM documentation markup language syntax?